admin_guide/scheduling/custom_scheduler.adoc

```diff
@@ -16,26 +16,99 @@ You can run multiple, custom schedulers alongside the default scheduler and conf
 
 To schedule a given pod using a specific scheduler, xref:admin-guide-sched-custom-deploy[specify the name of the scheduler in that pod specification].
 
-[[admin-guide-sched-custom-deploy]]
-== Deploying the Scheduler
-
-The steps below are the general process for deploying a scheduler into your cluster.
-
 [NOTE]
 ====
-Information on how to create/deploy a scheduler is outside the scope of this document. For an example, see link:https://github.com/kubernetes/kubernetes/tree/master/pkg/scheduler[*_plugin/pkg/scheduler_*] in the Kubernetes source directory.
+Information on how to create the scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation.
 ====
 
+[[admin-guide-sched-custom-package]]
+== Package the scheduler
+
+The general process for including a custom scheduler in your cluster
+involves creating an image and including that image in a deployment.
+
+. Package your scheduler binary into a container image. 
+
+. xref:../../creating_images/guidelines.html#creating-images-guidelines[Create a container image] containing the scheduler binary. 
++
+For example:
++
+----
+FROM <source>
```

Comment:
Suggestion: update "source" to "Parent image", "base image" or "source image". I checked quickly and I didn't see other examples of how we currently use this, but I was initially confused as to whether this meant source code, source repo, etc.